src/components/IvChart.tsx

```diff
@@ -79,30 +85,33 @@ const IvChart: React.FC<IvChartProps> = ({
   statIncreased,
   statDecreased,
   className,
-}) => (
-  <ChartContainer config={chartConfig} className={className}>
-    <RadarChart
-      data={stats.map((stat: Stat) => ({
-        stat: stat,
-        value: getStatIv(stat, ivs),
-      }))}
-    >
-      <ChartTooltip cursor={false} content={<ChartTooltipContent />} />
-      <PolarAngleAxis
-        dataKey="stat"
-        tick={(props: CustomTickProps) => (
-          <PokemonChartLabel
-            {...props}
-            statIncreased={statIncreased}
-            statDecreased={statDecreased}
-          />
-        )}
-      />
-      <PolarRadiusAxis className="hidden" domain={[0, 31]} />
-      <PolarGrid />
-      <Radar dataKey="value" fill="hsl(var(--primary))" fillOpacity={0.7} />
-    </RadarChart>
-  </ChartContainer>
-);
+}) => {
+  return (
+    <ChartContainer config={chartConfig} className={className}>
+      <RadarChart
+        data={stats.map((stat: Stat) => ({
+          labelData: `${stat},${getStatIv(stat, ivs)}`,
+          value: getStatIv(stat, ivs),
+        }))}
+        outerRadius="60%"
+      >
+        <ChartTooltip cursor={false} content={<ChartTooltipContent />} />
+        <PolarAngleAxis
+          dataKey="labelData"
+          tick={(props: CustomTickProps) => (
+            <PokemonChartLabel
+              {...props}
+              statIncreased={statIncreased}
+              statDecreased={statDecreased}
+            />
+          )}
+        />
+        <PolarRadiusAxis className="hidden" domain={[0, 31]} />
```

Comment:
I'm guessing this is also related to the MAX IV, let's use that constant here as well?

Reply:
Oh yeah totally missed that

Reply:
Fixed